.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Build & Test
+on:
+  push:
+    paths:
+      - "*.go"
+      - "*/*.go"
+      - "*/*/*.go"
+      - "*/*/*/*.go"
+      - "*/*/*/*/*.go"
+      - "*/*/*/*/*/*.go"

```

Comment:
Could we just do `"**.go"`?

Reply:
I'll have to check, but if I recall there was an issue with it not resolving correctly.